clang-tools-extra/docs/clang-tidy/checks/bugprone/implicit-widening-of-multiplication-result.rst

```diff
@@ -49,8 +49,7 @@ Options
 
    If the multiplication operands are compile-time constants (like literals or
    are ``constexpr``) and fit within the source expression type, do not emit a
-   diagnostic or suggested fix.  Only considers expressions where the source
-   expression is a signed integer type.  Defaults to ``false``.
+   diagnostic or suggested fix.  Defaults to ``false``.
```

Comment:
```suggestion
   diagnostic or suggested fix.  Defaults to `false`.
```

Single back-tick for option values. Please also fix other places.

Reply:
Should it be changed just for option values, or for all uses of double-backticks?  What is the semantic difference between single backticks and double ones? 

(Sorry, I've only ever used GitHub-flavored Markdown, which uses single backticks for inline code snippets, and triple ones for out-of-line snippets)

Reply:
Only for option values and names. Double back-ticks are for language constructs.